lib/rental.rb

```diff
@@ -5,7 +5,7 @@ def initialize(date, book, person)
     @date = date
     @book = book
     @person = person
-    book.rentals << self
-    person.rentals << self
+    book['rentals'] << self
+    person['rentals'] << self
   end
 end
```

Comment:
- [ ] Kindly make sure you fix this error when the rentals are being created. 

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/65068771/227258202-42faaa5e-392a-46c8-bc15-c9772783202c.png">

Reply:
Thanks @andersonlebon  for reviewing this PR,

Reply:
let's do it